configure/configure_whisk.yml

```diff
@@ -16,6 +16,6 @@ spec:
       restartPolicy: Never
       containers:
       - name: configure-openwhisk
-        image: danlavine/whisk_config:v1.5.6
+        image: mastermikeyboy/whisk_config:v1.6.2-dev
```

Comment:
Is this an intended switch to a different configure image? Or accidentally included in this PR?